### PR TITLE
travis: update go versions

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

**Comment by harshavardhana** [comment]
Thanks for the PR @dvrkps we are not upgrading to go1.9.x yet, there is a fix available already in this regard - https://github.com/minio/minio/pull/4872

**Review by harshavardhana — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -27,4 +27,4 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 
 go:
-- 1.8.3
+- 1.8.x
```
Comment (balamurugana) [inline .travis.yml:30]: One question is why we would need to use latest version of 1.8 than 1.8.3 as default.

Comment (harshavardhana) [reply .travis.yml:30]: This will pick the latest. So it could be 1.8.4 so it's an okay patch. Not needed strictly.

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**